assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -49,6 +50,7 @@ export const tests: Tests = {
     isActive: true,
     independent: true,
     seed: 5,
+    canRun: () => !getQueryParameter('contributionTypes'),
```

Comment:
you need to check for `defaultContributionType` too

I notice that we're calling it [`contributionType` in session storage](https://github.com/guardian/support-frontend/blob/master/assets/helpers/checkouts.js#L94) but `defaultContributionType` in the URL. Maybe they should both be `selectedContributionType` for clarity.